src/Page/Form.purs

```diff
@@ -204,6 +211,27 @@ component =
         , HP.id_ field.key
         , HE.onChecked (HE.input $ UpdateKey field.key <<< UserInput <<< Lynx.Data.Expr.Boolean)
         ]
+    Typeahead typeahead ->
+      HH.slot'
+        cp3
+        field.key
+        Typeahead.single
+        ( Typeahead.syncSingle
+          { itemToObject: \item -> fold do
+            { name: name', value: value' } <- toPair item
+            name <- toString name'
+            value <- toString value'
+            pure (Foreign.Object.singleton name value)
```

Comment:
`singleton` creates a key/value pair, and `Fuzzy` only searches the values, so this means `name` won't be used to search against. This should probably be:
```purescript
Foreign.Object.fromFoldable [ Tuple "name" name, Tuple "value" value ]
```

Reply:
I see. That doesn't seem to give us what we want then, right? I assume we're concerned that the `name` and `value` might be different. Isn't it possible that a search would find one and not the other? Also, which one should we want? Should we have an array of strings instead of an array of pairs?

Reply:
What we're likely to end up with is an array of json, and the user will define the key i.e. `name` or `value` to search through. They'll also define the same for rendering

Reply:
The [docementation for `Data.Fuzzy.match`](https://pursuit.purescript.org/packages/purescript-fuzzy/0.2.1/docs/Data.Fuzzy#v:match) might make it more clear what this `item -> Object String` is for

Reply:
Wait, I remember now. I was going off the assumption that we didn't actually care about the `name`, only the `value`. That's similar to dropdowns, where we don't actually care about the `name`, only the `value`. When we move on to the rest of the features for typeaheads (parsing JSON), we'll only have one field to search through anyway. Won't this will be duplicated information?

Reply:
In any case, I've made the changes. We can deal with the other stuff later.

Reply:
Well we'll want to let the users decide what they care about. For a type such as:
```purescript 
{ firstName :: String 
, last Name :: String
```
Most likely we would want to use both  values for search and display

Reply:
Well we'll want to let the users decide what they care about. For a type such as:
```purescript 
{ firstName :: String 
, last Name :: String
```
Most likely we would want to use both  values for search and display